corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -501,3 +538,19 @@ function intpad(x, n) {
     }
     return s;
 }
+
+/**
+ * Utility that gets the display options from a parent form of a question.
+ * */
+function _getDisplayOptions(question) {
+    var maxIter = 10; // protect against a potential infinite loop
+    var form = question.parent;
+
+    // logic in case the question is in a group or repeat or nested group, etc.
+    while (form.displayOptions === undefined && maxIter > 0) {
+        maxIter --;
```

Comment:
any chance we can throw an error or warning when this happens. i could see it being a very confusing bug